sdks/java/nexmark/src/main/java/org/apache/beam/sdk/nexmark/queries/WinningBids.java

```diff
@@ -192,6 +192,11 @@ public AuctionOrBidWindow decode(InputStream inStream)
     }
 
     @Override public void verifyDeterministic() throws NonDeterministicException {}
+
+    @Override
+    public boolean consistentWithEquals() {
+      return true;
```

Comment:
What do you think about `SUPER_CODER.consistentWithEquals() && ID_CODER.consistentWithEquals() && INT_CODER.consistentWithEquals()`? Not on a fast path, and makes the dependency quite explicit.

Reply:
This gets called when doing equality checks in StateTable maps, so it's actually relatively hot.  Changed to compute that value once and always return it.